user_docs/en/userGuide.md

```diff
@@ -2163,6 +2163,17 @@ This means that you do not have to scroll the display at the end of each line ev
 This may allow for more fluent reading of large amounts of text.
 It is disabled by default.
 
+##### Paragraph start markers {#BrailleParagraphStartMarkers}
+
+| . {.hideHeaderRow} |.|
```

Comment:
please move this table to the end of the settings group, as is the new standard described in userGuideStandards

Reply:
Done